Packs/SentryWire/Integrations/SentryWire/SentryWire.py

```diff
@@ -0,0 +1,370 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import urllib3
+import json
+import re
+from typing import Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    def __init__(self, address: str,
+                 username: str,
+                 password: str,
+                 port: int = 41395,
+                 version: str = "v3"):
+        super().__init__(base_url=f"https://{address}:{port}/{version}/",
+                         auth=(username, password),
+                         verify=False,
+                         proxy=False,
+                         headers={'Content-Type': "application/json"})
+        self._session.cookies.set(name='rest_token', value=self._get_auth(), domain=address)
+
+    def create_search(self, args: dict[str, Any]):
+        endpoint = 'fmsearch'
+        json_data = {
+            'rest_token': self._session.cookies.get('rest_token'),
+            'search_name': args.get('search_name'),
+            'search_filter': args.get('search_filter'),
+            'begin_time': args.get('begin_time'),
+            'end_time': args.get('end_time')
+        }
+        fmgroups = args.get('fmgroups')
+        max_packets = args.get('max_packets')
+        if fmgroups:
+            json_data['fmgroups'] = fmgroups
+        if max_packets:
+            json_data['max_packets'] = max_packets
```

Comment:
Cant you pass an empty string if no value was provided and then just get the args like above inside the json_data dict?
If not it will be more readable to use the if assignment. 
```suggestion
        if (fmgroups := args.get('fmgroups')):
            json_data['fmgroups'] = fmgroups

        if (max_packets := args.get('max_packets')):
            json_data['max_packets'] = max_packets

```

Reply:
Using an empty string or none will cause the search to fail which is why we have to handle optional settings here.